app/exports/tad_export.yml

```diff
@@ -40,6 +40,7 @@ custom_columns:
     - conditions_not_met
     - declined
     - declined_by_default
+    - deferred
```

Comment:
We currently include `offer_deferred` (on line 46), I was wondering how/if `deferred` was different?